src/test/java/bio/terra/integration/AzureIntegrationTest.java

```diff
@@ -435,6 +444,66 @@ private void populateOmopTable() throws Exception {
     dataRepoFixtures.updateSettings(steward, releaseSnapshotId, "omop/settings.json");
   }
 
+  @Test
+  public void testSnapshotCreateFromRequest() throws Exception {
+    populateOmopTable();
+    UUID snapshotRequestId = makeSnapshotAccessRequest().getId();
+    SnapshotSummaryModel snapshotSummaryByRequest = makeSnapshotFromRequest(snapshotRequestId);
+    String columnName = "datarepo_row_id";
+    List<Object> personSnapshotRows =
+        dataRepoFixtures
+            .retrieveSnapshotPreviewById(
+                steward, snapshotSummaryByRequest.getId(), "person", 0, 100, null, columnName)
+            .getResult();
+    List<Object> conditionOccurrenceSnapshotRows =
+        dataRepoFixtures
+            .retrieveSnapshotPreviewById(
+                steward,
+                snapshotSummaryByRequest.getId(),
+                "condition_occurrence",
+                0,
+                100,
+                null,
+                columnName)
+            .getResult();
+    List<Object> conceptSnapshotRows =
+        dataRepoFixtures
+            .retrieveSnapshotPreviewById(
+                steward, snapshotSummaryByRequest.getId(), "concept", 0, 100, null, columnName)
+            .getResult();
+    assertThat(personSnapshotRows.size(), is(23));
```

Comment:
Can you use `hasSize()` here?
```suggestion
    assertThat(personSnapshotRows, hasSize(23));
```